impl/src/main/java/com/okta/idx/sdk/impl/config/YAMLPropertiesSource.java

```diff
@@ -43,7 +43,8 @@ public YAMLPropertiesSource(Resource resource) {
     @Override
     @SuppressWarnings("unchecked")
     public Map<String, String> getProperties() {
-        try (InputStream in = resource.getInputStream()) {
+        try {
```

Comment:
Had to modify this - otherwise, JDK11 builds in Travis would fail with PMD error (false positive):

https://github.com/spotbugs/spotbugs/issues/1338